libkbfs/delayed_cancellation_test.go

```diff
@@ -102,6 +102,25 @@ func TestDelayedCancellationCleanupWhileNotEnabled(t *testing.T) {
 	}
 }
 
+func TestDelayedCancellationSecondEnable(t *testing.T) {
+	t.Parallel()
+
+	ctx, cancel := makeContextWithDelayedCancellation(t)
+	defer cancel()
+
+	err := EnableDelayedCancellationWithGracePeriod(ctx, 0)
+	if err != nil {
+		t.Fatalf("1st EnableDelayedCancellationWithGracePeriod failed: %v", err)
+	}
+	time.Sleep(5 * time.Millisecond) // make sure async ops sorts out
```

Comment:
hmm.. I can't think of a better way to do this here. So the goal of sleep is let it fail if grace period incorrectly starts in `EnableDelayedCancellationWithGracePeriod`, like before this PR. In this (incorrect) case, after the "async stuff" sorts out, the next call to `EnableDelayedCancellationWithGracePeriod` should fail. If we don't do this sleep, there's a chance that next line (2nd call to `EnableDelayedCancellationWithGracePeriod`) would happen the cancellation delayer updates its internal states, which make the 2nd call succeeds. Does this make sense?